src/examples/colors/color_grammar.py

```diff
@@ -75,11 +80,14 @@ def centroid(self) -> np.ndarray:
         major_term_map = {}
         for major_term in self.expressions:
             centroid = np.zeros(3)
-            
+
             for referent in major_term.meaning.referents:
-                centroid += np.array((referent.L, referent.a, referent.b)) * major_term.meaning._dist[referent.name]
+                centroid += (
+                    np.array((referent.L, referent.a, referent.b))
+                    * major_term.meaning._dist[referent.name]
+                )
             centroid /= len(major_term.meaning.referents)
```

Comment:
Noted! Yeah, the intention was to create a weighted average of the colors based off the probability distributions. I'll remove the division by the `len`, then. The centroid isn't currently being used, I thought it might be useful for potentially displaying color regions down the line.